crates/cheatnet/src/forking/state.rs

```diff
@@ -248,3 +252,25 @@ impl StateReader for ForkStateReader {
         ))
     }
 }
+
+fn generate_casm(sierra_contract_class: &Value) -> CasmContractClass {
+    let mut temp_sierra_file = Builder::new().tempfile().unwrap();
+    let _ = temp_sierra_file
+        .write(
+            serde_json::to_vec(sierra_contract_class)
+                .unwrap()
+                .as_slice(),
+        )
+        .unwrap();
```

Comment:
At this point, sierra_contract_class is a valid sierra code, right?

Reply:
Hmm, what do you mean by valid sierra code? 🤔

Reply:
Basically that none of those `.unwraps()` is going to fail

Reply:
The sierra_contract_class is just a `Value` here, so yes. None of those `.unwraps()` should fail

Reply:
It's serialized, so that's why I was concerned 👍